misc/test-clone-rm-kernel.sh

```diff
@@ -42,6 +42,9 @@ function do_clone_rm_kernel {
 
     mkdir -p /cephfs/destinations
     pushd /cephfs/destinations
+    if [ "$MAX_MDS" = d ]; then
```

Comment:
I don't think I understood this. Should it be `== d` ?

Reply:
No, see `test(1)`.

Reply:
Thanks! My Bash is rusty.